facebook-commerce-messenger-chat.php

```diff
@@ -22,6 +22,20 @@
 	 */
 	class WC_Facebookcommerce_MessengerChat {
 
+		/**
+		 * Page ID
+		 *
+		 * @var string
+		 */
+		private $page_id;
+
+		/**
+		 * JS SDK version
+		 *
+		 * @var string
+		 */
+		private $jssdk_version;
```

Comment:
This change breaks the unit tests.

Please verify that this is not accessed outside the class in any place as it happens in the unit tests.
If everything is fine then probably `setAccessible` in UT will do the trick.